docs/changes.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: Change Files
+hide_hero: true
+show_sidebar: false
+menubar: docs-menu
+---
+
+# Change Files
+
+A Notification of Change (NOC) is a non-dollar entry transmitted from a receiving (RDFI) to originating (ODFI) financial institution. These are sent in response to outdated or erroneous information in an initial entry. An NOC often occurs due to bank mergers or acquisitions that change account and/or routing numbers. If an RDFI sends an NOC, the ODFI will need to inform their originator promptly.
+
+The Standard Entry Class code for an NOC is "COR". There are a few possible reasons for an NOC, each defined by a "change code". The most common codes are `C01` for an incorrect account number and `C02` for an outdated routing number.
+
+NOCs are identified by an [Addenda98](https://pkg.go.dev/github.com/moov-io/ach?tab=doc#Addenda98) record on the EntryDetail with a [ChangeCode](https://pkg.go.dev/github.com/moov-io/ach?tab=doc#ChangeCode) that can be processed.
+
+### Processing
+
+The RDFI must send an NOC within two banking days of the original entry settlement date (to which the NOC is in response to).
+
+The ODFI is responsible for forwarding an NOC to the Originator within two banking days of the NOC settlement date. They must provide the Originator with the following information at a minimum:
+
+- Company name
+- Company identification
+- Company Entry description
+- Effective Entry date
+- DFI account number
+- Individual name/receiving company name
+- Individual indentification number/identification number

```

Comment:
What does `indentification` mean in this context? This may be a typo?

Reply:
Oh yeah, it's a typo, good catch